lib/active_mocker/generate.rb

```diff
@@ -93,7 +93,9 @@ def log_save(model)
   end
 
   def save_mock_file(klass_str, model)
-    File.open(File.join(Config.mock_dir, "#{model.class_name.underscore}_mock.rb"), 'w').write(klass_str)
+    File.open(File.join(Config.mock_dir, "#{model.class_name.underscore}_mock.rb"), 'w') do |f|
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.